src/turfs/processing.rs

```diff
@@ -168,7 +168,7 @@ fn _process_turf_start() -> Result<(), String> {
 		rayon::spawn(|| loop {
 			//this will block until process_turfs is called
 			let info = with_processing_callback_receiver(|receiver| receiver.recv().unwrap());
-			TASKS_RUNNING.fetch_add(1, Ordering::Release);
+			TASKS_RUNNING.fetch_add(1, Ordering::AcqRel);
```

Comment:
honestly just make this SeqCst, considering

Reply:
seqcst only does anything at all if you have multiple atomics doing seqcst. In this case it’s just acqrel